testing/oak_echo_service/proto/oak_echo_service.proto

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.echo;
+
+message EchoRequest {
+    bytes body = 1;
+}
+
+message EchoResponse {
+    bytes body = 1;
+}
+
+service EchoService {
+    // method_id: 1988
```

Comment:
What is number?

Reply:
Just a random number for testing.

Reply:
Should it be 0, since previously we were using 0,1,2 for method IDs in flatbuffers?